src/libraries/Errors.sol

```diff
@@ -4,6 +4,8 @@ pragma solidity >=0.8.22;
 import { IERC721Metadata } from "@openzeppelin/contracts/token/ERC721/extensions/IERC721Metadata.sol";
 import { UD60x18 } from "@prb/math/src/UD60x18.sol";
 
+import { ISablierRecipient } from "../interfaces/ISablierRecipient.sol";
```

Comment:
Why don't we remove this import and instead use `address` in the errors below?

```solidity
error SablierV2Lockup_AllowToHookIncorrectImplementation(address recipient);
error SablierV2Lockup_AllowToHookZeroCodeSize(address recipient);
```

Reply:
https://github.com/sablier-labs/v2-core/pull/951/commits/c71ad185a7e2bc5eb245ef749d54ef7f6058ed83